components/Graph.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useEffect } from 'react';
+import {
+  CartesianAxis,
+  CartesianGrid,
+  Legend,
+  Line,
+  LineChart,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { LegendLineGraphIcon } from '@openclimatefix/nowcasting-ui.icons.icons';
+import useSWR, { Fetcher } from 'swr';
+
+interface forecastDataPointProps {
```

Comment:
Convention is to capitalize all interface definitions. Do this for the others as well.

```suggestion
interface ForecastDataPointProps {
```